config/routes.rb

```diff
@@ -14,4 +14,6 @@
   namespace :admin do
     resources :dashboard, only: [:index]
   end
+
+  get '*unmatched_route', to: 'application#not_found'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.